keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -353,6 +363,38 @@ Contains a unique identifier for the containing cluster.
 
 **Reusing cluster names**: Since an `id.k8s.io ClusterProperty` has no restrictions on whether or not a ClusterProperty can be repeatable, if a cluster unregisters from a ClusterSet it is permitted under this standard to rejoin later with the same `id.k8s.io ClusterProperty` it had before. Similarly, a *different* cluster could join a ClusterSet with the same `id.k8s.io ClusterProperty` that had been used by another cluster previously, as long as both do not have membership in the same ClusterSet at the same time. Finally, two or more clusters may have the same `id.k8s.io ClusterProperty` concurrently (though they **should** not; see "Uniqueness" above) *as long as* they both do not have membership in the same ClusterSet.
 
+#### Property: `network.k8s.io`
+
+Contains an identifier representing the network for the cluster.
+
+
+##### Uniqueness
+
+*   The identifier **may not** exist (as its only applicable for multi-network scenario) and **need not** be unique
+
+##### Lifespan
+
+*   The identifier, if exists, **should** be immutable for the lifespan of a ClusterSet membership.
+
+
+##### Contents
+
+*   The identifier **should** be a valid RFC-1123 DNS label [as described for object names in the Kubernetes docs](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names).
```

Comment:
I don't like "should" here.  It's pointless.  How will this be used?  I assume it will be primarily interpreted by implementations of things like MCS  and maybe compared(opaquely) between clusters, but not interpreted by arbitrary clients?

As such, I don't think we need to spec anything about character set.  Implementations will take "should" as advisory or they will end up doing pointless transformations.

Reply:
I agree. I can change this to `string`, I think the intention is to keep it compact and also human readable.

Reply:
I agree that "compact and human readable" is a meaningful SHOULD  :)